dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/config_text_input.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useState, useEffect } from 'react';
+import { EuiFieldText, EuiTitle, EuiSpacer, htmlIdGenerator } from '@elastic/eui';
+
+interface InputFieldProps {
+  name: string;
+  title: string;
+  currentValue: string;
+  handleInputChange: (value: string) => void;
+}
+
+export const TextInputFieldItem: React.FC<InputFieldProps> = ({
+  name,
+  title,
+  currentValue,
+  handleInputChange,
+}) => {
+  const [fieldValue, setFieldValue] = useState<string>(currentValue);
+
+  useEffect(() => {
+    setFieldValue('');
```

Comment:
The logic here is a little bit confused to me, seems like you always set it to be '' even when there's value for currentValue, and can we maybe name it with a different name as currentName I feel not really a fit here for this component, not sure if it makes sense to you.